src/pages/Admin/Admin.tsx

```diff
@@ -456,22 +459,24 @@ const Admin: React.FC = () => {
   }
 
   return (
-    <WithNavigationBar fullscreen>
-      <div className="admin-dashboard">
-        <div className="page-container page-container_adminview">
-          <div className="page-container-adminsidebar">
-            <VenueList selectedVenueId={venueId} roomIndex={queryRoomIndex} />
-          </div>
-          <div className="page-container-adminpanel">
-            {venueId ? (
-              <VenueDetails venueId={venueId} roomIndex={queryRoomIndex} />
-            ) : (
-              <>Select a venue to see its details</>
-            )}
+    <RelatedVenuesProvider venueId={venueId}>
+      <WithNavigationBar fullscreen>
```

Comment:
What do we need related venues for here in the admin panel? I wouldn't expect it should be needed at all.

My guess is that this was added due to something in `WithNavigationBar` needing it. But depending on what that is, I expect it ALSO probably has no place being in the admin panel. The navigation bar in the admin panel should be SUPER slim/basic, and I know @denisdimitrov had a PR for refactoring this to slim it right down. No idea if that ever got merged or if it's stuck in 'PR rot hell'.

Reply:
Apparently it's maybe for this? https://github.com/sparkletown/sparkle/pull/1713#discussion_r665029238

Reply:
I think the reason was https://github.com/sparkletown/sparkle/pull/1620/files#r655862063 . Can't remember clearly, I think the hook was there, but needed the provider

Reply:
Makes sense.